README.md

```diff
@@ -70,61 +88,69 @@ classifier = classifier.train_intent(train)
 predicted = classifier.test_intent(test)
 res = pd.DataFrame(list(zip(test.intents, predicted)), columns=['true', 'predicted'])
 ```
+
 If you need to configure **stratification**, use the `stratification` parameter (defaults to `"intents"` and uses the intents in the dataset as stratification basis; whatever _else_ you pass along has to conform to `sklearn.model_selection.train_test_split(stratify=)`:
+
 ```python
 train, test = dataset.train_test_split(test_size=0.25, random_state=0, stratification=None)    # deactivate stratification (sklearn default for train_test_split)
 ```
 
+To compare your own vendor or algorithm to existing vendors in this package, you can
+write a Vendor Subclass for your vendor, and possibly a dataloader function. Feel free
+to share your implementation using this repo. Similarly, fixes and extensions for the
+existing vendors are always welcome.
+
 ### Logging
 
 Most of the code uses python logging to report its progress. To get logs printed out
-to console or Jupyter notebook, a logger needs to be configured, before the nlutests 
+to console or Jupyter notebook, a logger needs to be configured, before the nlutests
 code. Usually, log messages are on INFO level. This can be configured like this:
 
-````python
+```python
 import logging
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 logger.addHandler(logging.StreamHandler())
-````
-
+```
 
 ### Concepts / Architecture
 
-* **Vendors**  
-The [`vendors`](/nlubridge/vendors/) module implements standardized interfaces to the 
-  specific vendors. A specific `Vendor` instance is in charge of dealing with converting 
-  the data to the required format, uploading data to the cloud if applicable, training 
+- **Vendors**\
+  The [`vendors`](/nlubridge/vendors/) module implements standardized interfaces to the
+  specific vendors. A specific `Vendor` instance is in charge of dealing with converting
+  the data to the required format, uploading data to the cloud if applicable, training
   models and making predictions.
-  
-* **Datasets**  
-The [`datasets`](/nlubridge/datasets/) module provides a standard interface to 
+
+- **Datasets**\
+  The [`nlu_dataset`](/nlubridge/datasets/) module provides a standard interface to
```

Comment:
Link ist falsch. Muss 
`The [`nlu_dataset`](/nlubridge/nlu_dataset/) module provides a standard interface to`
heißen.